README.md

```diff
@@ -141,49 +177,41 @@ the snippet above with `<arg value="-F"/>`.
 
 To run diktat to check/fix code style - run `$ mvn antrun:run@diktat`.
 
-### Use the new diktat-maven-plugin
+## Run with Gradle
+### Use diktat-gradle-plugin
+You can see how the plugin is configured in our project for self-checks: [build.gradle.kts](build.gradle.kts).
```

Comment:
same as above, I guess. Add to examples or ask @aktsay6

Reply:
As discussed, we will do it under #264 after gradle plugin is released.